README.md

```diff
@@ -87,6 +87,16 @@ $lib->setToken($token->access_token)
 ### Authenticated
 
 
+#### Single User Applications
+
+If you only ever plan on interacting with a single user account, feel free to hard code (or preferably configure) your access tokens into your app.
+
+```php
+$lib->setToken($access_token);
+```
```

Comment:
Definitely keep the example. This looks good. 

Just fill out our CLA (keeps the code licensing clear): developer.vimeo.com/cla and I'll merge this in.